LICENSE

```diff
@@ -0,0 +1,256 @@
+Skip to content
```

Comment:
please remove this file, I have created one

Reply:
ok